api/spec/src/main.tsp

```diff
@@ -8,6 +8,7 @@ import "./auth.tsp";
 import "./errors.tsp";
 import "./types.tsp";
 
+import "./app";
```

Comment:
This feels like it should be "apps", not "app".

Reply:
As we keep the Go pkg singular for now, keeping this one too.